doc/source/serve/api/index.md

```diff
@@ -44,31 +44,44 @@ This is fixed by added custom filename mappings in `source/conf.py` (look for "a
    serve.handle.RayServeSyncHandle
 ```
 
-#### Advanced APIs
+### Running Applications
 
 ```{eval-rst}
 .. autosummary::
    :nosignatures:
    :toctree: doc/
 
-   serve.get_replica_context
-   serve.get_multiplexed_model_id
-   serve.get_app_handle
-   serve.get_deployment_handle
+   serve.run
+   serve.delete
+   serve.start
+   serve.shutdown
+   serve.status
 ```
 
-### Running Applications
+### Configuring System-level Options
 
 ```{eval-rst}
 .. autosummary::
    :nosignatures:
    :toctree: doc/
 
-   serve.run
-   serve.delete
    serve.start
-   serve.shutdown
-   serve.status
+   serve.config.ProxyLocation
+   serve.config.HTTPOptions
```

Comment:
Nice! Could we add `serve.config.gRPCOptions` as well?

Reply:
On a similar note, could we add `schema.gRPCOptionsSchema` in the config schema section?
<img width="758" alt="Screen Shot 2023-08-31 at 3 25 19 PM" src="https://github.com/ray-project/ray/assets/15851518/d9949347-d567-4302-a76d-475f244693d2">

Reply:
leaving these to @GeneDer